### Hypothesis Testing Question 3

Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions.

|                |     East    |     West    |     North    |     South    |
|----------------|-------------|-------------|--------------|--------------|
|     Males      |     50      |     142     |     131      |     70       |
|     Females    |     550     |     351     |     480      |     350      |


#### Hypothesis Definition
H0 : All proportions are equal.

H1 : Not all proportions are equal.

In [2]:
import numpy as np
import pandas as pd
from scipy.stats import chi2_contingency

In [3]:
data1= pd.read_csv("BuyerRatio.csv")
data1.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [4]:
arr = np.array([[50,142,131,70],[435,1523,1356,750]]) #Dimensional Array
arr

array([[  50,  142,  131,   70],
       [ 435, 1523, 1356,  750]])

In [5]:
chi2_contingency(arr) # OP is in the format : chi2,p-value,dof,expected

(1.595945538661058,
 0.6603094907091882,
 3,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

#### Result
p-value = 0.6603 > 0.05. Therefore, we accept the Null Hypotheis that all proportions are equal.i.e. Independence of categorical variables. Thus, male-female buyer rations are similar across regions and are not related

